subprojects/ide-native/src/main/java/org/gradle/ide/xcode/tasks/GenerateXcodeWorkspaceFileTask.java

```diff
@@ -32,12 +32,16 @@
  */
 @Incubating
 public class GenerateXcodeWorkspaceFileTask extends XmlGeneratorTask<XcodeWorkspaceFile> {
-    private DefaultXcodeWorkspace xcodeWorkspace;
+    private Provider<Iterable<File>> xcodeProjectLocations;
```

Comment:
I used `FileCollection` instead with setter and getter as it seems to align better with other similar usage and we only need to set the value in one place.